### DISCUSSION TITLE
disable compress

### DISCUSSION THREAD

**Discussion by pesian** [jlesage/docker-firefox#206] [Q&A] [answered]
hi
how to set 0 compress  when load page by default
try to changed index.html and and rfb.js  but its not workings

**Comment by jlesage** [comment]
The `Compression Level` slider from the left pop-up menu should do the job.  The value is saved into a cookie so it is kept across visits from the same browser.

**Reply by pesian** [reply]
tnx for your answer
yes its work  but  i want to set **default** value from 2 to 0 because When the processor is weak, the browsing becomes very slow

**Reply by jlesage** [reply, marked as answer]
Currently there is no built-in way to change the default value (without user interaction).

Other ways to do it:

- Set the compression via the URL.  For example: `http://192.168.1.1:5800/?compression=0`.
- In `/opt/noVNC/app/ui.js`, change the line `UI.initSetting('compression', 2);` to `UI.initSetting('compression', 0);`

**Reply by pesian** [reply]
thnx a lot 
you are awesome
